adapters/oidc/servlet-filter/pom.xml

```diff
@@ -30,6 +30,16 @@
     <name>Keycloak Servlet Filter Adapter Integration</name>
     <description/>
 
+    <properties>
+        <keycloak.osgi.export>
+            org.keycloak.adapters.servlet.*
+        </keycloak.osgi.export>
+        <keycloak.osgi.import>
+            org.keycloak.*;version="${project.version}",
```

Comment:
Does this mean that all other keycloak dependencies of this Adapter are also deployed as OSGI bundles?

Reply:
Yes, that's correct. The complete list of the deps is the following:

* keycloak-adapter-core
* keycloak-adapter-spi
* keycloak-authz-client
* keycloak-common
* keycloak-core
* keycloak-osgi-thirdparty (pre-4.2.0)
* keycloak-servlet-adapter-spi

All the modules, except for keycloak-servlet-adapter-spi, have been previously published as OSGi bundles. This PR fills the missing piece.